### DISCUSSION TITLE
Change plugin priority UX/UI design

### DISCUSSION THREAD

**Discussion by NikoMuffin** [Flow-Launcher/Flow.Launcher#506] [Ideas]
The priority field blends in with other non-clickable fields making it difficult for new or even experienced users to realize one can change the priority!!
![image](https://user-images.githubusercontent.com/1836362/118755889-9566bd00-b81e-11eb-86a1-3722cfd0ce53.png)

I honestly did not notice this until someone responded to my Github question.  I think this design needs to be revised as plugin priority is a huge deal and I'm pretty sure others are not noticing it as well.

 I am proposing making Plugin Priority a bit more obvious
![image](https://user-images.githubusercontent.com/1836362/118755759-5173b800-b81e-11eb-9c15-5ba9a0c8e87f.png)

**Comment by taooceros** [comment]
I am not quite sure about how shall we handle the new design, because currently only this line is controlled by Flow, and the rest of the panel is controlled by plugin.

**Comment by NikoMuffin** [comment]
> I am not quite sure about how shall we handle the new design, because currently only this line is controlled by Flow, and the rest of the panel is controlled by plugin.


I will defer to the team's decision.  This to me is not a high priority since I know where it is now, but others may suffer my same fate (unless documentation is done very well to highlight it perhaps?).

OK to close if the dev(s) don't feel it would add to the app.

**Comment by jjw24** [comment]
> I am not quite sure about how shall we handle the new design, because currently only this line is controlled by Flow, and the rest of the panel is controlled by plugin.

Any chance we can do that on the same line/panel of the plugin title?

We are edging close to the necessity of enlarging the settings page overall size lol

**Comment by jjw24** [comment]
I have updated the documentation on this:
https://flow-launcher.github.io/docs/#/usage-tips
